src/gui/dialogs/DialogSelftestFans.cpp

```diff
@@ -10,9 +10,9 @@ DialogSelftestFans::DialogSelftestFans()
     : IDialogMarlin()
     , text_fan_test(this, Rect16(col_0, 40, WIZARD_X_SPACE, 22), is_multiline::no, is_closed_on_click_t::no, _("Fan test"))
     , progress(this, Rect16(col_0, 62, WIZARD_X_SPACE, 8), COLOR_LIME, COLOR_BLACK)
-    , text_extruder_fan(this, Rect16(col_0, 74, col_0_w, 22), is_multiline::no, is_closed_on_click_t::no, _("Hotend fan"))
+    , text_extruder_fan(this, Rect16(col_0, 74, col_0_w, 22), is_multiline::no, is_closed_on_click_t::no, _("Print fan"))
```

Comment:
rename `text_extruder_fan `and `text_print_fan`
to `text_print_fan` and `text_hotend_fan`

Reply:
changes made - extruder fan renamed to hotend fan and extruder icon to hotend icon to be in accordance with standard fan naming:
fan1 = hotend fan + hotend fan Icon
fan0 = print fan and print fan icon